repositories.yaml

```diff
@@ -171,15 +171,15 @@ libraries:
     url: https://github.com/mozilla/gecko-dev
     metrics_files:
       - browser/base/content/metrics.yaml
-      - gfx/metrics.yaml
-      - toolkit/components/glean/metrics.yaml
-      - toolkit/components/processtools/metrics.yaml
       - dom/media/metrics.yaml
       - dom/metrics.yaml
+      - gfx/metrics.yaml
       - netwerk/metrics.yaml
       - netwerk/protocol/http/metrics.yaml
+      - toolkit/components/glean/metrics.yaml
+      - toolkit/components/processtools/metrics.yaml
     ping_files:
-      - toolkit/components/glean/pings.yaml
+      - toolkit/components/telemetry/pings.yaml
```

Comment:
🤔 This is wrong I think. `toolkit/components/telemetry/pings.yaml` should be the one in `firefox_desktop`: https://searchfox.org/mozilla-central/source/toolkit/components/glean/metrics_index.py#70-72